components/Avatar/avatar.html

```diff
@@ -0,0 +1,126 @@
+      <nav class="navbar">
+        <!-- <span class="navbar__logo"> </span> -->
+        <a href="/index.html" class="navbar__logo">LazyCss</a>
+
+        <ul class="navbar__list">
+          <li><a href="/index.html">Home </a></li>
+          <li><a href="#">Docs </a></li>
+          <li>
+            <a
+              href="https://github.com/KuldeepSinghRathore/lazycss/tree/developement"
+              >Github
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <section class="grid--section">
+      <aside class="aside">
+        <ul class="aside__list">
+          <li class="component-links">
+            <a href="/components/Button/button.html">Button</a>
+          </li>
+          <li class="component-links">
+            <a href="/components/Avatar/avatar.html">Avatar</a>
+          </li>
+          <li class="component-links">
+            <a href="/components/Alert/alert.html">Alert</a>
+          </li>
+          <li class="component-links">
+            <a href="/components/Badge/badge.html">Badge</a>
+          </li>
+          <li class="component-links">
+            <a href="/components/Card/card.html">Card</a>
+          </li>
+          <li class="component-links">
+            <a href="/components/Image/image.html">Image</a>
+          </li>
+          <li class="component-links">
+            <a href="/components/Input/input.html">Input</a>
+          </li>
+          <li class="component-links">
+            <a href="/components/Lists/lists.html">Lists</a>
+          </li>
+          <li class="component-links">
+            <a href="/components/Navigation/navigation.html">Navigation</a>
+          </li>
+          <li class="component-links">
+            <a href="/components/Modal/modal.html">Modal</a>
+          </li>
+          <li class="component-links">
+            <a href="/components/Rating/rating.html">Rating</a>
+          </li>
+          <!-- <li class="component-links">
+            <a href="/components/Grid/grid.html">Grid</a>
+          </li>
+          <li class="component-links">
+            <a href="/components/Text/text.html">Text</a>
+          </li>
+          <li class="component-links">
+            <a href="/components/Toast/toast.html">Toast</a>
+          </li>
+          <li class="component-links"></li> -->
+        </ul>
+      </aside>
+      <main>
+        <img
+          class="avatar avatar--150"
+          src="https://miro.medium.com/fit/c/262/262/1*iCHrefA0Ig4R5mu6M56z8Q.jpeg"
+          alt="Tanay bhaiya"
+        />
+        <pre> 
+    <code  class ="language-html">
+    &lt;img class=&quot;avatar avatar--150&quot; 
+    src=&quot;https://miro.medium.com/fit/c/262/262/1*iCHrefA0Ig4R5mu6M56z8Q.jpeg&quot;
+    alt=&quot;Tanay bhaiya&quot; /&gt; 
+    </code>
+    </pre>
+        <img
+          class="avatar avatar--100"
+          src="https://miro.medium.com/fit/c/262/262/1*iCHrefA0Ig4R5mu6M56z8Q.jpeg"
+          alt="Tanay bhaiya"
+        />
```

Comment:
check the class naming here , this could be "avatar-sm" or "avatar-md" this could make sense than "avatar-100 or 150"